_pages/maiabee.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: maia-bee
+pageUrl: maia-bee.github.io/make-a-website
+timestamp: 2017-01-24
+---
```

Comment:
@maia-bee just remove this file as it doesn't contain the dash in your user handle